src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponentSpec.scala

```diff
@@ -119,41 +119,26 @@ class SubmissionComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers
     }
   }
 
-  "WorkflowComponent" should "let you modify Workflows within a submission" in withDefaultTestDatabase {
```

Comment:
this diff is really stupid looking. i deleted the old and only test for WorkflowComponent and wrote two new ones.